### PR TITLE
Homework16

### PR DESCRIPTION
**Pull request by mariankatysovska**

### CODE REVIEW

**Comment by mariankatysovska** [comment]
Но у меня почему-то Test Failed. Вообще не открывается ссылка в Chrome. Не могу понять в чем проблема.

**Comment by Berichicko** [comment]
Please, add Homework16Test in TestNG.xml to run tests

For example

image

File: src/test/java/Homework16Test.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+public class Homework16Test extends BaseTest{
+
+
+    @Test
+    public  static void Navigation () {
+        ChromeOptions options = new ChromeOptions();
+        options.addArguments("--remote-allow-origins=*");
+        options.addArguments("--disable-notifications");
+
+        WebDriver driver = new ChromeDriver();
```
Comment (Berichicko) [inline src/test/java/Homework16Test.java]: Please finish the homework!You got this problem, please correct it. 

image

HINT

`WebDriver driver = new ChromeDriver(options);`

File: src/test/java/Homework16Test.java [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.testng.annotations.Test;
+
+
+
+public class Homework16Test extends BaseTest{
+
+
+    @Test
+    public  static void Navigation () {
```
Comment (Berichicko) [inline src/test/java/Homework16Test.java]: Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.

For example `public  static void navigation ()`

**Pull request closed without merging**